packages/rpcsvc_proto/project.bri

```diff
@@ -15,7 +15,7 @@ const source = Brioche.download(
 export default function rpcsvcProto(): std.Recipe<std.Directory> {
   return std.runBash`
     ./configure --prefix=/
-    make -j16
+    make -j "$(nproc)"
```

Comment:
Just to notice here, I didn't set the env variable OMP_NUM_THREADS to override nproc command output. I think it's not necessary for this package.